<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMaterialRenamer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="aspectsBox">
        <property name="title">
         <string>Aspects</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_15">
           <property name="text">
            <string>&amp;Form</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_16">
           <property name="text">
            <string>&amp;Composition</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBox_19">
           <property name="text">
            <string>&amp;Brushwork</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_18">
           <property name="text">
            <string>&amp;Texture</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBox_20">
           <property name="text">
            <string>&amp;Matte</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_17">
           <property name="text">
            <string>&amp;Expression</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_14">
           <property name="text">
            <string>&amp;Color</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="checkBox_21">
           <property name="text">
            <string>&amp;GraphicDesign</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="uncheckAllTagsButton">
        <property name="text">
         <string>&amp;Uncheck all tags</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="newTagsBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>New tags</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_9"/>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="addTagButton">
        <property name="text">
         <string>&amp;Add tag</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="inspiringBox">
        <property name="title">
         <string>INSPIRING</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_23">
           <property name="text">
            <string>&amp;Inspiring</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="newTagName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>New tag name</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="creatorsBox">
        <property name="title">
         <string>Creators</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_17">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_24">
           <property name="text">
            <string>&amp;Studios</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="nsfwBox">
        <property name="title">
         <string>NSFW</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_22">
           <property name="text">
            <string>&amp;NSFW</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Renamer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Image</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <widget class="QLineEdit" name="creatorsName">
           <property name="placeholderText">
            <string>John Smith</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Creator's name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QFrame" name="imageFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QLabel" name="imageFrameLabel">
              <property name="text">
               <string>Image goes here</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>Directories</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0">
          <widget class="QLineEdit" name="sourceDirectory">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="loadFilesButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Load files</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Source directory</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Files</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>New filename:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="currentFileName">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 105, 140);</string>
           </property>
           <property name="text">
            <string>[Load in the images first]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Current filename:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="newFileName">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(46, 107, 80);</string>
           </property>
           <property name="text">
            <string>[New filename goes here]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="title">
         <string>Navigation/Submit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="previousImageButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Previous Image</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="nextImageButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Next Image</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="renameButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Rename</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_9">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <widget class="QGroupBox" name="mediumsBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Mediums</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QCheckBox" name="tag_2D">
           <property name="text">
            <string>&amp;2D</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="tag_3D">
           <property name="text">
            <string>&amp;3D</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="tag_Traditional">
           <property name="text">
            <string>&amp;Traditional</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="tag_Photograph">
           <property name="text">
            <string>&amp;Photograph</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="stylesBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="title">
         <string>Styles</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="2" column="0">
          <widget class="QCheckBox" name="tag_SemiRealistic">
           <property name="text">
            <string>&amp;SemiRealistic</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>&amp;VeryStylized</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>&amp;Stylized</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="tag_Realistic">
           <property name="text">
            <string>&amp;Realistic</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="tag_HyperRealistic">
           <property name="text">
            <string>&amp;HyperRealistic</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBox_13">
           <property name="text">
            <string>&amp;Abstract</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="subjectsBox">
        <property name="title">
         <string>Subjects</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_7">
           <property name="text">
            <string>&amp;Nature</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="checkBox_11">
           <property name="text">
            <string>&amp;Buildings</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="text">
            <string>&amp;Vehicles</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="checkBox_10">
           <property name="text">
            <string>&amp;Robots</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBox_9">
           <property name="text">
            <string>&amp;Weapons</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_6">
           <property name="text">
            <string>&amp;Objects</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>&amp;Characters</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBox_8">
           <property name="text">
            <string>&amp;Technology</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_5">
           <property name="text">
            <string>&amp;Landscapes</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="checkBox_12">
           <property name="text">
            <string>&amp;Caricatures</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>74</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
